pkg/plugins/discovery.go

```diff
@@ -103,3 +94,10 @@ func readPluginJSONInfo(name, path string) (*Plugin, error) {
 
 	return &p, nil
 }
+
+func pluginPaths(base, name, ext string) []string {
+	return []string{
+		filepath.Join(base, name+ext),
```

Comment:
if there is both do you want the plugin.sock to win against plugin/plugin.sock? or return an error?